demos/python_demos/human_pose_estimation_3d_demo/README.md

```diff
@@ -23,15 +23,19 @@ The demo workflow is the following:
 > **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_Converting_Model_General.html).
 ## Prerequisites
 
-Before running it is necessary to build `pose_extractor` module. Your system should have installed:
-* Python 3.5 (or above)
-* CMake 3.10 (or above)
-* C++ Compiler (g++ or MSVC)
-
-To build `pose_extractor` module, please run in command line:
-`python setup.py build_ext`
```

Comment:
I'm not sure we should remove that approach. As @vladimir-dudnik said, we don't want make python users mess with C++ build tools.

Reply:
It's a fair point, but:

1. I would really prefer not to maintain two build systems.
2. The `setup.py`-based system delegates to CMake anyway. So it's not like it's substantively different.

Reply:
`setup.py` removes a need to manually building a command line to compile the lib (like `build_demos_msvc.bat`). Alternative solution could be adding `ENABLE_PYTHON` key to `build_demos*` scripts. I'm just trying to fix a concern that it would be inconvenient for python users to use `cmake` manually. On the other hand for now we can agree that this is a hybrid demo and you should be familiar with both python and C++/cmake to use it.

So the new question is should `build_demos*` forward `ENABLE_PYTHON` key.

Reply:
Are you suggesting to add an option to the `build_demos*` scripts to set `ENABLE_PYTHON`?

Reply:
Yes

Reply:
OK, I've done this for `build_demos.sh`. I'll do it for the Windows script later.

Reply:
> I'll do it for the Windows script later.

And done.